maestro-inspectors/maestro-amqp-inspectors/src/main/java/org/maestro/inspector/amqp/writers/RouteLinkInfoWriter.java

```diff
@@ -0,0 +1,91 @@
+package org.maestro.inspector.amqp.writers;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.maestro.common.inspector.types.RouterLinkInfo;
+import org.maestro.common.writers.InspectorDataWriter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+
+public class RouteLinkInfoWriter implements InspectorDataWriter<RouterLinkInfo>, AutoCloseable {
+    private static final Logger logger = LoggerFactory.getLogger(RouteLinkInfoWriter.class);
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+    private BufferedWriter writer;
+    private CSVPrinter csvPrinter;
+
+    public RouteLinkInfoWriter(final File logDir, final String name) throws IOException {
+        File outputFile = new File(logDir, name + ".csv");
+
+        System.out.println(outputFile.getPath());
+
+        writer = Files.newBufferedWriter(Paths.get(outputFile.getPath()), Charset.defaultCharset());
+        csvPrinter = new CSVPrinter(writer, CSVFormat.DEFAULT
+                .withHeader("Timestamp", "Name", "LinkType", "LinkDir", "Identity", "OperStatus",
```

Comment:
I want to find better solution for this. Maybe some kind of constant array or something like this This is creepy I think.